app/graphql/types/pairing_type.rb

```diff
@@ -1,17 +1,19 @@
 module Types
   class PairingType < Types::BaseObject
     field :id, ID, null: false
-    field :pairer, Types::UserType, null: false
-    field :pairee, Types::UserType, null: false
+    field :pairer, Types::UserType, null: true
+    field :pairee, Types::UserType, null: true
     field :date, String, null: true
     field :time, String, null: true
     field :notes, String, null: true
```

Comment:
Curious about this part: I was under the impression `pairer`, `date`, and `time` would need to be required. Did this change fix something else?